azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -265,10 +266,12 @@ public FlowRunnerManager(final Props props,
     this.cleanerThread = new CleanerThread();
     this.cleanerThread.start();
 
-    if (this.azkabanProps.getBoolean(ConfigurationKeys.AZKABAN_POLL_MODEL, false)) {
+    if (DispatchMethod.getDispatchMethod(azkabanProps
+        .getString(Constants.ConfigurationKeys.AZKABAN_EXECUTION_DISPATCH_METHOD, "PUSH"))
+        == DispatchMethod.POLL) {
```

Comment:
nit: is it possible to encapsulate this in a method or maybe create a private attribute `dispatchMethod`?

Reply:
Thank you for reviewing the PR. I have take care of comments.